[quant][graphmode] Remove `fold_prepack` for now

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #30553 [quant][graphmode] `insert_quant_dequant` pass support shared class types
* **#30909 [quant][graphmode] Remove `fold_prepack` for now**
* #30860 [quant][graphmode][refactor] Factor out getInvokedMethod in `InsertQuantDeQuantHelper`
* #30859 [quant][graphmode][refactor] getQParams return a dictionary of qparams

Summary:
`fold_prepack` doesn't work anymore after we change `scale`, 'zero_point`
to be attributes, but since the freeze API is coming up, I don't want to
spend time to make this work since this will be thrown away later.

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18864537](https://our.internmc.facebook.com/intern/diff/D18864537)

jerryzh168 committed:
[quant][graphmode] Remove `insert_prepack_unpack` and `fold_prepack` for now

Summary:
`fold_prepack` doesn't work anymore after we change `scale`, 'zero_point`
to be attributes, but since the freeze API is coming up, I don't want to
spend time to make this work since this will be thrown away later.

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM commented on torch/quantization/_quantize_script.py:
```diff
@@ -79,13 +79,6 @@ def convert_script(model, inplace=False):
         model = model.copy()
     torch._C._jit_pass_insert_quant_dequant(model._c, 'forward', True)
-    if 'fbgemm' in torch.backends.quantized.supported_engines:
-        torch._C._jit_pass_insert_prepack_unpack(model._c)
```
I think our fusion patterns expect `unpack` nodes to be present and with this change we are no longer inserting them. How does it work?

jerryzh168 replied:
oh, sorry, I'll keep prepack_unpack

jerryzh168 committed:
Update on "[quant][graphmode] Remove `fold_prepack` for now"


Summary:
`fold_prepack` doesn't work anymore after we change `scale`, 'zero_point`
to be attributes, but since the freeze API is coming up, I don't want to
spend time to make this work since this will be thrown away later.

Test Plan:
.

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18864537](https://our.internmc.facebook.com/intern/diff/D18864537)

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Looks good!
